src/games/strategy/triplea/delegate/BattleDelegate.java

```diff
@@ -1610,5 +1635,7 @@ public IBattle getCurrentBattle() {
   boolean m_needToRecordBattleStatistics;
   boolean m_needToCheckDefendingPlanesCanLand;
   boolean m_needToCleanup;
+  boolean m_needToDoRockets;
+  RocketsFireHelper rocketHelper;
```

Comment:
If you host a game with this new branch, and join with current master, you get this error on startup:
`java.lang.NoSuchFieldError: m_needToDoRockets`

```
java.lang.NoSuchFieldError: m_needToDoRockets
	at games.strategy.triplea.delegate.BattleDelegate.saveState(BattleDelegate.java:175)
	at games.strategy.engine.framework.GameDataManager.writeDelegates(GameDataManager.java:245)
	at games.strategy.engine.framework.GameDataManager.saveGame(GameDataManager.java:222)
	at games.strategy.engine.framework.GameDataManager.saveGame(GameDataManager.java:209)
```